src/javascript/crypto/e2e/openpgp/pgpmime/utils.js

```diff
@@ -29,12 +29,24 @@ goog.require('e2e.openpgp.pgpmime.Text');
 goog.require('e2e.openpgp.pgpmime.types.Entity');
 
 goog.require('goog.array');
-goog.require('goog.asserts');
 goog.require('goog.object');
 goog.require('goog.string');
 
-var invalidMsg = 'Invalid MIME format';
 
+/**
+ * A message that is displayed when invalid MIME messages are encountered.
+ * @const
+ * @private
+ */
+var INVALID_MESSAGE_ = 'Invalid MIME format';
```

Comment:
Make it a class constant, not a stray variable:

e2e.openpgp.....INVALID_MESSAGE_ =  ''